### PR TITLE
*: add slow-query doc

### PR DESCRIPTION
**Pull request by coocood**

### CODE REVIEW

**Comment by coocood** [comment]
@shenli @zz-jason PTAL

File: op-guide/tidb-config-file.md [hunk 1]
```diff
@@ -48,6 +48,12 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 + 默认: false
 + 如果设置为 true，那么日志里面将不会输出时间戳。
 
+### `slow-query-file`
+
++ 慢查询日志的文件名。
++ 默认: ""
++ 设置后，慢查询日志单独会输出到该文件。
```
Comment (cocoyi17) [inline op-guide/tidb-config-file.md]: 单独会 -> 会单独

File: sql/slow-query.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+---
+title: 慢查询日志
+category: user guide
+---
+
+# 慢查询日志
+
+一个不合理的 SQL 语句会导致整个集群压力增大，响应变慢。对于这种问题，我们需要用慢查询日志来定位有问题的语句，解决性能问题。
```
Comment (cocoyi17) [inline sql/slow-query.md]: 一个 -> 一条

File: sql/slow-query.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+### 获取
+
+通过在 TiDB 的日志文件上 grep SLOW_QUERY 这个关键字, 可以得到执行时间超过 [slow-threshold](../op-guide/tidb-config-file.md#slow-threshold) 的语句日志。
```
Comment (cocoyi17) [inline sql/slow-query.md]: SLOW_QUERY -> `SLOW_QUERY`
关键字, ->关键字，

File: sql/slow-query.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
+---
+title: 慢查询日志
+category: user guide
+---
+
+# 慢查询日志
+
+一个不合理的 SQL 语句会导致整个集群压力增大，响应变慢。对于这种问题，我们需要用慢查询日志来定位有问题的语句，解决性能问题。
+
+### 获取
```
Comment (yikeke) [inline sql/slow-query.md]: Add more details in this heading:
获取 -> 获取日志

File: sql/slow-query.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+表示这个语句在 TiKV 的处理时间之和，因为数据会并行的发到 TiKV 执行，这个值可能会超过 cost_time。
+
+#### wait_time
+
+表示这个语句在 TiKV 的等待时间之和，因为 TiKV 的 coprocessor 线程数是有限的，当所有的 coprocessor
+线程都在工作的时候，请求会排队，当队列中有某些请求耗时很长的时候，后面的请求的等待时间都会增加。
+
+#### backoff_time
+
+表示语句遇到需要重试的错误，在重试前等待的时间，有很多种可能，常见的原因有遇到了 lock，region 分裂,
```
Comment (yikeke) [inline sql/slow-query.md]: In a Chinese text, use a Chinese back-sloping comma (顿号) to separate items in a list:
lock，region 分裂, -> lock、region 分裂、

File: sql/slow-query.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+total_keys:1869712 processed_keys:1869710 succ:true con:3 user:root@127.0.0.1
+txn_start_ts:402329674704224261 database:test table_ids:[31],index_ids:[1],
+sql:select count(c) from sbtest1 use index (k_1)
+```
+
+### 字段解析
+
+#### cost_time
+
+表示执行这个语句花费的时间。只有执行时间超过 slow-threshold
```
Comment (yikeke) [inline sql/slow-query.md]: There are some redundant line breaks (换行符) in this document, please remove them.

File: sql/slow-query.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+#### processed_keys
+
+表示 coprocessor 处理的 key 的数量。相比 total_keys，processed_keys 不包含
+mvcc 的旧版本和 mvcc delete 标记。如果 processed_keys 和 total_keys 相差
+很大，说明旧版本比较多。
+
+#### succ
+
+请求是否执行成功
```
Comment (yikeke) [inline sql/slow-query.md]: Keep consistent with the above descriptions below headings:
请求是否执行成功 -> 表示请求是否执行成功。

File: sql/slow-query.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+
+#### con
+
+connection ID，即 session ID, 可以用类似 `con:3 ` 的关键字在日志中 grep 出 session ID 为 3 的日志。
```
Comment (yikeke) [inline sql/slow-query.md]: Please add "表示" at the beginning of every description sentences below and add the period (句号) if needed.

**Comment by coocood** [comment]
@yikeke 
PTAL

**Pull request merged**